game/ui/general/agentsheet.cpp

```diff
@@ -9,6 +9,7 @@
 #include "framework/renderer.h"
 #include "game/state/gamestate.h"
 #include "game/state/rules/battle/damage.h"
+#include <string>
```

Comment:
It's not necessary to include this here.
If you need to convert numbers to string use some of the functions in `library/strings.h`. You can also use formatting (tinyformat syntax): `format("%d/%d", modified_stats.x, current_stats.x)`

Reply:
I removed `if` expressions. It will be two of stats separated by slash `"Health: 99/99"` or smth like this.